scripts/populate_db/populate_clients.py

```diff
@@ -0,0 +1,154 @@
+import requests
+
+from fractal_server.app.schemas import ApplyWorkflowCreate
+from fractal_server.app.schemas import ApplyWorkflowRead
+from fractal_server.app.schemas import DatasetCreate
+from fractal_server.app.schemas import DatasetRead
+from fractal_server.app.schemas import ProjectCreate
+from fractal_server.app.schemas import ProjectRead
+from fractal_server.app.schemas import ResourceCreate
+from fractal_server.app.schemas import ResourceRead
+from fractal_server.app.schemas import TaskCreate
+from fractal_server.app.schemas import TaskRead
+from fractal_server.app.schemas import WorkflowCreate
+from fractal_server.app.schemas import WorkflowRead
+from fractal_server.app.schemas import WorkflowTaskCreate
+from fractal_server.app.schemas import WorkflowTaskRead
+
+
+BASE_URL = "http://localhost:8000"
+CREDENTIALS = dict(username="admin@fractal.xy", password="1234")  # nosec
+
+
+class SimpleHttpClient:
+    base_url: str
+
+    def __init__(self, base_url, credentials):
+        self.base_url = base_url
+
+        response = requests.post(
+            f"{self.base_url}/auth/token/login/",
+            data=credentials,
+        )
+
+        self.bearer_token = response.json().get("access_token")
+
+    def make_request(self, endpoint, method="GET", data=None):
+        headers = {"Authorization": f"Bearer {self.bearer_token}"}
+        url = f"{self.base_url}/{endpoint}"
+
+        try:
+            if method == "GET":
+                response = requests.get(url, headers=headers)
+            elif method == "POST":
+                response = requests.post(url, json=data, headers=headers)
+            elif method == "PATCH":
+                response = requests.patch(url, json=data, headers=headers)
+            elif method == "DELETE":
+                response = requests.delete(url, headers=headers)
+            else:
+                raise ValueError(f"Unsupported HTTP method: {method}")
+
+            return response
+
+        except requests.RequestException as e:
+            print(f"Request failed: {e}")
+            return None
+
+
+class New:
```

Comment:
New method: `wait_for_all_jobs`.

if called as superuser, it can use directly `/admin/job/`, with a query param for status.

within a while loop, this can be blocking until all jobs are failed or done